File: src/main/java/com/gomes800/bus_location_backend/domain/BusLocation.java
package com.gomes800.bus_location_backend.domain;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class BusLocation {

    @JsonProperty("ordem")
    private String order;
    private String latitude;
    private String longitude;
    @JsonProperty("datahora")
    private String dateTime;
    @JsonProperty("velocidade")
    private String speed;
    @JsonProperty("linha")
    private String line;
    @JsonProperty("datahoraenvio")
    private String dateTimeShipping;
    @JsonProperty("datahoraservidor")
    private String dateTimeServer;

}

File: app/src/main/java/com/example/bus_location_android/domain/BusLocation.kt
package com.example.bus_location_android.domain

import com.fasterxml.jackson.annotation.JsonProperty

data class BusLocation(
    @JsonProperty("ordem")
    val order: String,
    val latitude: String,
    val longitude: String,
    @JsonProperty("datahora")
    val dateTime: String,
    @JsonProperty("velocidade")
    val speed: String,
    @JsonProperty("linha")
    val line: String,
    @JsonProperty("datahoraenvio")
    val dateTimeShipping: String,
    @JsonProperty("datahoraservidor")
    val dateTimeServer: String
)